### DISCUSSION TITLE
Building of NextJs app is failing

### DISCUSSION THREAD

**Discussion by tarunyadav1** [wevm/wagmi#2185] [Q&A] [answered]
After installing Rainbow and wagmi and setting up the boilerplate code. While `yarn build` gives the following error. 

These are the version that I am using.
`"wagmi": "^0.12.10"`
`"@rainbow-me/rainbowkit": "^0.12.8",`

![image](https://user-images.githubusercontent.com/56935638/232120264-d72e0e52-2d12-4739-af5e-104a99ee1b2f.png)

**Comment by tmm** [comment, marked as answer]
What version of TypeScript are you using? wagmi [supports 4.7.4](https://wagmi.sh/react/typescript) and above.

**Reply by tarunyadav1** [reply]
I was using `4.5.5` version. Upgrading the TypeScript version resolved the issue. Thank you!
